test/skimNtuple2018.cpp

```diff
@@ -47,8 +49,16 @@
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/format.hpp>
 #include <Math/VectorUtil.h>
+#include <Math/LorentzVector.h>
+#include <Math/PxPyPzM4D.h>
+
+// DNN
```

Comment:
I know I had told you to comment it out for the moment, but can you uncomment this DNN part?
https://github.com/camendola/KLUBAnalysis/pull/34/files#r417865527